### PR TITLE
Add coverage for the Schema, Link, Example, ...

### PR DESCRIPTION
**Pull request by turkeylurkey**
ExternalDocumentation and Extension annotations

### CODE REVIEW

File: tck/src/main/java/org/eclipse/microprofile/openapi/apps/airlines/resources/ReviewResource.java [hunk 1]
```diff
@@ -189,9 +189,7 @@ public Response getReviewByUser(
             description = "username of the user for the reviews",
             required = true,
             in = ParameterIn.PATH,
-            content = @Content(
-                examples = @ExampleObject(
-                    value = "bsmith")))
```
Comment (arthurdm) [inline tck/src/main/java/org/eclipse/microprofile/openapi/apps/airlines/resources/ReviewResource.java]: can we keep the current `content` field as well?    That way we're not decreasing coverage on other annotations.

File: tck/src/main/java/org/eclipse/microprofile/openapi/apps/airlines/resources/ReviewResource.java [hunk 2]
```diff
@@ -222,9 +220,7 @@ public Response getReviewByUser(
                 description = "name of the airlines for the reviews",
                 required = true,
                 in = ParameterIn.PATH,
-                content = @Content(
-                    examples = @ExampleObject(
-                        value = "Acme Air")))
```
Comment (arthurdm) [inline tck/src/main/java/org/eclipse/microprofile/openapi/apps/airlines/resources/ReviewResource.java]: same thing here - I think we can keep the `content`.

**Comment by arthurdm** [comment]
thanks for PR @turkeylurkey - I just made 2 small comments, if you can please address those.

**Comment by turkeylurkey** [comment]
Ok, I restored those @Contents.

**Pull request merged**